ros_ign_bridge/README.md

```diff
@@ -1,2 +1,284 @@
 # This is a shim package
 For [ros_gz_bridge](https://github.com/gazebosim/ros_gz/tree/ros2/ros_gz_bridge)
+
```

Comment:
I'd rather not maintain two README files. Can we just keep the one in `ros_gz_bridge`?

Reply:
Yeah, I don't care about updating the README anymore lol.